onnx/test/parser_test.py

```diff
@@ -119,6 +119,205 @@ def test_parse_function_with_attributes(self) -> None:
         model = onnx.parser.parse_model(input)
         checker.check_model(model)
 
+    def test_parse_afline_grid(self) -> None:
```

Comment:
Seems better to move this test to a different file, like `test_affine_grid_function.py`, it is not a parser test, it just uses the parser.

Reply:
this test is for this PR development only - quick validation of the function body. Once all tests are passing, I will remove this test. the test body in this test will be moved to tensor/def.cc.